app/trip/create/page.tsx

```diff
@@ -1,20 +1,27 @@
 'use client'
 import { Heading, Box, Container, useColorModeValue } from '@chakra-ui/react'
 import { Header, Footer } from '@/components/navigation'
+import { TripForm } from '../components'
 
 export default function CreateTripPage() {
   const bg = useColorModeValue('white', 'gray.800')
   const color = useColorModeValue('black', 'gray.300')
+
   return (
     <>
       <Header />
       <Box as="main" minH="100vh" bg={bg} color={color}>
         <Container
-          maxW={{ base: '100%', lg: 'container.xl' }}
+          maxW={{ base: '100%', lg: 'container.sm' }}
```

Comment:
When the screen size is tablet size, it still takes up 100% of the width, and the input fields appear too wide. Perhaps consider adding container.sm starting from the medium size, like the example below:
`maxW={{ base: '100%', md: 'container.sm' }}`
<div style="display: flex; justify-content: flex-start; align-items: flex-start;">
  <img width="300" alt="Screenshot 1" src="https://github.com/tabi-memo/frontend/assets/70562492/b333270a-825d-478b-b77d-12f51d17de44">
  <img width="300" alt="Screenshot 2" src="https://github.com/tabi-memo/frontend/assets/70562492/634652eb-5840-4164-9b63-7da63d344bd8">
</div>